main_window/packet_parser.py

```diff
@@ -34,9 +52,23 @@
 }
 DEVICE_TYPE_TO_ID = {y: x for (x, y) in ID_TO_DEVICE_TYPE.items()}
 
+# Matches enum EventId in FLARE: radio.h
+EVENT_IDS = {
+    0x00: "LAUNCH",
+    0x01: "STAGE SEPARATION",
+    0x02: "MACH LOCK ENTER",
+    0x03: "MACH LOCK EXIT",
+    0x04: "APOGEE",
+    0x05: "DROGUE DEPLOY",
+    0x06: "MAIN DEPLOY",
+    0x07: "LANDED"
+}
+
```

Comment:
If possible, might want to use the enum "DataEntryValues" in the data_entry_ids.py